<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveSnapshotDialog</class>
 <widget class="QDialog" name="SaveSnapshotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Screenshot Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="selectedViewOnly">
     <property name="toolTip">
      <string>&lt;html&gt;When checked, only the active view will be saved, otherwise all views will be saved in the image as they are layed out presently.&lt;/html&gt;</string>
     </property>
     <property name="statusTip">
      <string>&lt;html&gt;When checked, only the active view will be saved, otherwise all views will be saved in the image as they are layed out presently.&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;When checked, only the active view will be saved, otherwise all views will be saved in the image as they are layed out presently.&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Save only selected view</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Select resolution for the image to save&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="width"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="height"/>
     </item>
     <item>
      <widget class="QToolButton" name="lockAspect">
       <property name="toolTip">
        <string>Lock aspect </string>
       </property>
       <property name="statusTip">
        <string>Lock aspect </string>
       </property>
       <property name="whatsThis">
        <string>Lock aspect </string>
       </property>
       <property name="text">
        <string>L</string>
       </property>
       <property name="icon">
        <iconset resource="../pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqOctreeData16.png</normaloff>:/pqWidgets/Icons/pqOctreeData16.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;b&gt;Select image quality (if applicable)&lt;/b&gt;&lt;br/&gt;0 - low quality, 100 - high quality</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="pqIntRangeWidget" name="quality" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;b&gt;Override Color Palette&lt;/b&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="palette">
     <item>
      <property name="text">
       <string>Current Palette</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Helvetica'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Stereo Mode (if applicable)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="stereoMode">
     <item>
      <property name="text">
       <string>No Stereo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Red-Blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Interlaced</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Checkerboard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Side By Side Horizontal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Left Eye Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Right Eye Only</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqIntRangeWidget</class>
   <extends>QWidget</extends>
   <header>pqIntRangeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pqComponents.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>SaveSnapshotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>SaveSnapshotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
